TestAssets/TestProjects/LightupClient/Program.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.IO;
+using System.Reflection;
+
+namespace LightupClient
+{
+    public static class Program
+    {
+        public static void Main(string[] args)
+        {
+            Assembly asmGreet = null;
+            
+            try
+            {
+                asmGreet = Assembly.Load(new AssemblyName("LightupLib"));
+            }
+            catch(Exception ex)
+            {
+                throw new FileNotFoundException("Exception: Failed to load the lightup assembly!", ex);
+            }
+
+            if (asmGreet == null)
+            {
+                throw new FileNotFoundException("Failed to load the lightup assembly!");
+            }
+            else
+            {
+                Console.WriteLine("Lightup assembly successfully loaded!");
```

Comment:
Maybe we should call into the `Greet.Hello` method and write its result to the console.  That proves that the assembly is loaded and code can be executed in it.